easybuild/easyconfigs/m/MPICH/MPICH-3.2-GCC-7.2.0-2.29.eb

```diff
@@ -0,0 +1,16 @@
+name = 'MPICH'
+version = '3.2'
```

Comment:
@rubendibattista Any reason you're not updating to `3.2.1`?

Reply:
Not really. I just edited the one that was there with a previous GCC. 

I'm planning to PR `gmpich-2018a` with GCC-7.3 and MPICH 3.2.1. How does it work with toolchains? Should I do something special or wait?

Reply:
It usually makes sense to check whether there are any software updates, bug fixes can always be useful.
I see you've issued #5883 for MPICH 3.2.1, we can merge both, that's fine.